apps/frontend/src/layout/partials/UserProfile.tsx

```diff
@@ -110,12 +112,21 @@ export const UserProfile = () => {
                             return (
                                 <DropdownMenuItem
                                     className="px-3 py-2"
-                                    key={item?.name}
+                                    key={item?.id}
                                     onClick={() => {
-                                        navigate(`/${item?.slug}`);
+                                        switchWorkspace(
+                                            convertToAuthOrganization(item) as AuthOrganization,
+                                            item?.projects?.edges![0]?.node as Project,
+                                        );
                                     }}
                                 >
-                                    <NavItem text={item?.name} leftIcon={<AcronynmBox text={item?.name ?? ""} />} />
+                                    <button className="flex w-full cursor-pointer items-center gap-2 overflow-x-hidden rounded capitalize text-intg-text ">
+                                        <span>
+                                            <AcronynmBox text={item?.name ?? ""} />
+                                        </span>
+                                        <span>{item?.name}</span>
+                                    </button>
+                                    {/* <NavItem text={item?.name} leftIcon={<AcronynmBox text={item?.name ?? ""} />} /> */}
```

Comment:
@afolasope if we uncomment the `NavItem` and remove the `button` element it was replaced with, the content would still look the similar, was this intentional?

Reply:
I have removed it the button element